custom_components/tahoma/cover_devices/tahoma_cover.py

```diff
@@ -92,6 +96,10 @@ def current_cover_tilt_position(self):
 
     async def async_set_cover_position_low_speed(self, **kwargs):
         """Move the cover to a specific position with a low speed."""
+        _LOGGER.warning(
+            "This service will removed in the next releases. Use instead the low speed switch."
```

Comment:
Should we just mention a release, or even better a month? This service will be removed in September 2021 e.g.

Reply:
Thx for the suggestion. I didn’t know how to formulate. 
>This service will be removed in September 2021.
sounds good. @vlebourl what do you think?